### ISSUE TITLE
 The reward of "CartPole-v1"  running with dppo does not rise

### ISSUE THREAD

**Issue by aipla999** [seolhokim/DistributedRL-Pytorch-Ray#7]
Hello, code is very clear ,but I encountered some problems while running the code.

There are no errors ,but the reward of "CartPole-v0"  running with **_dppo_** does not rise.
I think there may be bugs in  agents/runners/learners/dppo_learner.py.

On line 13（ "`gradients_ids = ( [agent.run.remote() for agent in actors]）`" ），
the agent.brain's gradient has been calculated,but catually it is None.
Therefore, all brain's weights can not be updated.

Besides, On line 14 (  "`while len(gradients_ids)`:" ) , whether the length of the `gradients_ids` will be a constant, and then lead to wrong results.

**Comment by seolhokim** [comment]
All you mentioned correctly work for me. `gradients_ids` is the variable to identify the completed agent. But I found some bugs related with cuda. It stops training by error, and only the test agent works. plz check the latest commit. All things are work use_cuda is False version.
